packages/cozy-stack-client/src/DocumentCollection.spec.js

```diff
@@ -462,4 +466,33 @@ describe('DocumentCollection', () => {
       expect(resp.data).toHaveDocumentIdentity()
     })
   })
+
+  describe('changes', () => {
+    const collection = new DocumentCollection('io.cozy.todos', client)
+    beforeEach(() => {
+      client.fetchJSON.mockReturnValueOnce(
+        Promise.resolve({
+          last_seq: 'new-seq',
+          results: [
+            { doc: { _id: '1', done: false, label: 'Fetch changes' } },
+            { doc: null },
+            { doc: { _id: '_design/view' } },
+            { doc: { _id: '2', _deleted: true, label: 'Refactor code' } }
+          ]
+        })
+      )
+    })
+
+    it('should call the right route', async () => {
+      const changes = await collection.fetchChanges('my-seq')
+      expect(client.fetchJSON).toHaveBeenCalledWith(
+        'GET',
+        '/data/io.cozy.todos/_changes?include_docs=true&since=my-seq'
+      )
+      expect(changes).toEqual({
+        newLastSeq: 'new-seq',
+        documents: [{ _id: '1', done: false, label: 'Fetch changes' }]
+      })
+    })
+  })
```

Comment:
Can we also add a test for `includeDesign` & `includeDeleted` ?

Reply:
Thanks Quentin, I was just lazy ><